### DISCUSSION TITLE
Definition of the initial population for the genetic algorithm by the user

### DISCUSSION THREAD

**Discussion by zsbeheshti** [PyGithub/PyGithub#2495] [Q&A]
Dear all,

I am using Python's pypi genetic algorithm 1.0.2 and I want to run this genetic algorithm with my default initial population. How can I do this?

And is it possible that, for example, out of 70 populations, I introduce only one and the genetic algorithm produces the other 69?

Regards

**Comment by EdmundGoodman** [comment]
Hi! I think you may be asking your question on the wrong GitHub repository?

At a guess, the repository you are looking for might be [https://github.com/rmsolgi/geneticalgorithm](https://github.com/rmsolgi/geneticalgorithm), not `PyGithub` - perhaps try making an issue/discussion other there instead?

**Comment by EnricoMi** [comment]
Yes, does not look related to this repo.
